_search-plugins/search-relevance/index.md

```diff
@@ -98,4 +98,62 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
 {: .note}
 
 You cannot save a given comparison for future use, so Compare Search Results is not suitable for systematic testing.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Comparing search results with the OpenSearch Intelligent Ranking plugin
+
+The OpenSearch Intelligent Ranking plugin takes search results returned from OpenSearch and processes them using the Amazon Kendra intelligent enterprise search service. The Amazon Kendra service leverages semantic search capabilities to re-rank OpenSearch results, considering the context of the OpenSearch query and relevant documents from OpenSearch.
+
+To use the OpenSearch Intelligent Ranking plugin, you must first set up the Amazon Kendra service. To get started, see [Amazon Kendra](https://aws.amazon.com/kendra/). For plugin setup instructions, see [OpenSearch Intelligent Ranking plugin](https://docs.aws.amazon.com/kendra/latest/dg/opensearch-rerank.html).
```

Comment:
When will this link work?

Reply:
Should work on 11/28.

Reply:
btw, both links work now: 
https://docs.aws.amazon.com/kendra/latest/dg/what-is-kendra.html
https://aws.amazon.com/kendra/